saleor/graphql/order/mutations/orders.py

```diff
@@ -62,6 +90,59 @@ class Meta:
         model = models.Order
 
 
+class OrderUpdateShippingInput(graphene.InputObjectType):
+    shipping_method = graphene.ID(
+        description='ID of the selected shipping method.',
+        name='shippingMethod')
+
+
+class OrderUpdateShipping(BaseMutation):
+    class Arguments:
+        id = graphene.ID(
+            required=True, name='order',
+            description='ID of the order to update a shipping method.')
+        input = OrderUpdateShippingInput(
+            description='Fields required to change '
+                        'shipping method of the order.')
+
+    class Meta:
+        description = 'Updates a shipping method of the order.'
+
+    @classmethod
+    @permission_required('order.manage_orders')
+    def mutate(cls, root, info, id, input):
+        errors = []
+        order = cls.get_node_or_error(info, id, errors, 'id', Order)
+
+        if not input['shipping_method']:
+            if order.is_shipping_required():
+                cls.add_error(
+                    errors, 'shippingMethod',
+                    'Shipping method is required for this order.')
+                return OrderUpdateShipping(errors=errors)
+            order.shipping_method = None
+            order.shipping_price = ZERO_TAXED_MONEY
+            order.shipping_method_name = None
+            order.save()
+            return OrderUpdateShipping(order=order)
+
+        method = cls.get_node_or_error(
+            info, input['shipping_method'], errors,
+            'shipping_method', ShippingMethod)
+        clean_order_update_shipping(order, method, errors)
+        if errors:
+            return OrderUpdateShipping(errors=errors)
+
+        order.shipping_method = method
+        order.shipping_price = method.get_total_price(info.context.taxes)
+        order.shipping_method_name = method.name
+        order.save()
+        return OrderUpdateShipping(order=order)
+
+    order = graphene.Field(
```

Comment:
Class fields should appear at the top of the class declaration, above the metaclasses and methods. From what I see our codebase lacks consistency in this area so it may be a good idea to do the overall refactoring, but that's a separate PR. Let's fix at least this line in this PR.